tests/ui/no_mangle_with_rust_abi.rs

```diff
@@ -25,7 +25,7 @@ fn rust_abi_multiline_function_really_long_name_to_overflow_args_to_multiple_lin
     0
 }
 
-// Must not run on functions that explicitly opt in to Rust ABI with `extern "Rust"`
+// Must not run on functions that explicitly opt into Rust ABI with `extern "Rust"`
```

Comment:
`in to`/`into` has been quite contentious, happy to drop...

Reply:
Should be 'opt in to using the Rust ABI' if you want to be more formal.